env/stage.json

```diff
@@ -0,0 +1 @@
+
```

Comment:
@oscarmild 
**Is this file for staging server?**

Reply:
@fromgopi 
Yes this for staging server

Reply:
@oscarmild  is staging server up ?

Reply:
@fromgopi No server team is in the process of setting up the stating server

Reply:
@oscarmild ok.